containers/README.md

```diff
@@ -0,0 +1,129 @@
+# GTSAM Containers
+
+- container files to build images
+- script to push images to a registry
+- instructions to pull images and run containers
+
+## Dependencies
+
+- a container engine
+
+Below we use [podman](https://podman.io), but you should be able to replace any `podman` calls to a Docker-compatible CLI such as [`docker`](https://www.docker.com/) or [`nerdctl`](https://github.com/containerd/nerdctl).
```

Comment:
Could we just use docker to make it easy for folks?

Reply:
sure, I'll switch `podman` calls out to `docker`.

Reply:
Done, removed mentions to alternative container tools.

I figured that if someone cares enough about using a different tool than Docker, they'll put in the effort to install such a tool, figure out the correct commands to build/pull/push images, and the commands to run containers. (Which itself shouldn't be a big deal since tools like podman and nerdctl have a Docker-compatible CLI.)